store-and-retrieve-data/use-storagehub-sdk/create-a-bucket.md

```diff
@@ -74,120 +39,155 @@ async function run() {
+    
+    Once this check passes, the `createBucket` extrinsic is called, and `bucketId` along with the `txReceipt` are returned back to the `index.ts` file from where you will trigger this helper method.
 
-Finally, you can call the `createBucket()` function using the `storageHubClient`, including the previously gathered parameters and the `isPrivate` flag that determines the bucket’s privacy.
+### Call the Create Bucket Helper Method
 
-```ts title="// **PLACEHOLDER FOR STEP 6: CREATE BUCKET**"
---8<-- 'code/store-and-retrieve-data/use-storagehub-sdk/create-a-bucket/create-a-bucket.ts:create-bucket'
-```
+Now that you've extracted all the bucket creation logic into its own method, let's update the `index.ts` file.
+
+1. Replace the placeholder `// **PLACEHOLDER FOR STEP 2: CREATE BUCKET**` with the following code:
+
+    ```ts title="index.ts // **PLACEHOLDER FOR STEP 2: CREATE BUCKET**"
+    --8<-- 'code/store-and-retrieve-data/use-storagehub-sdk/create-a-bucket/create-a-bucket.ts:create-bucket'
+    ```
+
+    !!! note
+        You can also get a list of all your created buckets within a certain MSP using the `mspClient.buckets.listBuckets()` function. Make sure you are authenticated before triggering this function.
+
+
+2. Execute the `createBucket` method by running the script:
+
+    ```bash
+    ts-node index.ts
+    ```
+
+    The response should look something like this:
+
+    --8<-- 'code/store-and-retrieve-data/use-storagehub-sdk/create-a-bucket/output-02.html'
 
-!!! note
-    You can also get a list of all your created buckets within a certain MSP using the `mspClient.buckets.listBuckets()` function. Make sure you are authenticated before triggering this function.
 
 ## Check if Bucket is On-Chain
 
-The last step is to verify that the bucket was created successfully on-chain and to confirm its stored data.
+The last step is to verify that the bucket was created successfully on-chain and to confirm its stored data. Just like with the `createBucket` method you can extract all the bucket verification logic into its own `verifyBucketCreation` method. 
 
-```ts title="// **PLACEHOLDER FOR STEP 7: VERIFY BUCKET**"
---8<-- 'code/store-and-retrieve-data/use-storagehub-sdk/create-a-bucket/create-a-bucket.ts:verify-bucket'
-```
+1. Add the following code in your `bucketOperations.ts` file:
+    
+    ```ts title="bucketOperations.ts"
+    --8<-- 'code/store-and-retrieve-data/use-storagehub-sdk/create-a-bucket/bucketOperations.ts:verify-bucket'
+    ```
+
+2. Update the `index.ts` file to trigger the helper method you just implemented:
+
+    ```ts title="index.ts // **PLACEHOLDER FOR STEP 3: VERIFY BUCKET**"
+    --8<-- 'code/store-and-retrieve-data/use-storagehub-sdk/create-a-bucket/create-a-bucket.ts:verify-bucket'
+    ```
 
 The response should look something like this:
```

Comment:
I would nest the terminal output under the last step so it's consistent with the other sections